proto/palomachain/paloma/evm/set_smart_contract_deployer_proposal.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package palomachain.paloma.evm;
+
+option go_package = "github.com/palomachain/paloma/x/evm/types";
+
+message SetSmartContractDeployerProposal {
+  string title = 1;
+  string summary = 2;
+  string chainReferenceID = 3;
+  string deployerAddress = 4;
```

Comment:
What about turning this into a batch proposal? That way, we can set up multiple chains with one vote.